dss/storage/checkout/cache_flow.py

```diff
@@ -0,0 +1,37 @@
+import json
+import os
+from dss.storage.hcablobstore import FileMetadata
+
+"""
+These functions assist with the caching process, the lifecycle policies are set to delete files within
+the checkout buckets;
+For AWS object tagging is used to mark files for deletion: TagSet=[{uncached:True}]
+For GCP object classes are used to indicate what is to be cached: STANDARD_STORAGE (MULTI-REGIONAL) are cached
+See MetaData Caching RFC for more information (Google Docs)
```

Comment:
add a link to the google doc if you're going to mention it.

Reply:
make sure it's a view only or comment only link to the doc.

Reply:
Should we be mentioning our RFC documents in the code?

Reply:
is that google doc something that is internal? perhaps we can update the [RFC](https://github.com/HumanCellAtlas/dcp-community/blob/rfc-metadata-caching/rfcs/text/0000-rfc-metadata-caching.md) and point to that?

Reply:
I think copying the relevant information into a docstring at the top of `cache_flow.py` would make sense.

Reply:
the doc is part of the HCA which is all public, but we don't want public write access.

Reply:
I see no problem with including it in the comments.